.github/workflows/docker_build.yml

```diff
@@ -13,6 +13,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - 'v*'
```

Comment:
couldn't we just do `on: release`?

Reply:
Maybe. I'm not seeing that in the documentation for https://github.com/docker/metadata-action

Reply:
hmm, you're right. i guess this is fine for now